float64[] polygonCoordinates
float64 penalty
int64 epochValidFrom
int64 epochValidTo
